dcp_prototype/backend/wrangling/migrations/common/dataset_metadata.py

```diff
@@ -50,6 +50,18 @@ def process(self):
         self.process_dissociation_protocol()
         self.process_library_preparation_protocol()
         self.process_enrichment_protocols()
+        self.process_cell_suspension()
+        self.process_organoid()
+        self.process_cell_line()
+
+        # NOTE: this may not be the proper way to get the biosample_category.
+        # This need to be reviewed and checked.
```

Comment:
I think you can use this request: 

http GET https://service.explore.data.humancellatlas.org/repository/projects?size=50 --body | jq -r '.hits[] | "\(.entryId)\t\"\(.projects[].projectShortname)\"\t\(.samples[].sampleEntityType | join(","))"'

Let me know if that doesn't work!

Reply:
The other option might be just to hard code it because (at least for now) this code is only used to convert the DCP 1.0 data. In case you decide that option, you could probably re-create the `constants.py` file in `common/` and add the information in based on this spreadsheet:

https://docs.google.com/spreadsheets/d/1u6T37eij_Qm0cY2eyKbuAvJhZfGsIQUHDMFhczDEwLA/edit#gid=0

Reply:
I updated to the the rest api.